Classes/Dao/PageSeoDao.php

```diff
@@ -40,6 +42,20 @@ class PageSeoDao extends Dao
     protected $pageTreeView;
 
     /**
+     * Extended options set via plugin.metaseo.extensibility
+     * @var array
+     */
+    protected $extensibilityOptions;
+
+    public function __construct()
```

Comment:
constructor is not necessarily executed with test suite.

There should be a setter which would let the test suite inject a mocked object.